stdlib/universe/schema_mutators.go

```diff
@@ -206,13 +206,15 @@ func NewDropKeepMutator(qs flux.OperationSpec) (*DropKeepMutator, error) {
 }
 
 func (m *DropKeepMutator) checkKeepColumns(tableCols []flux.ColMeta) error {
-	if m.KeepCols != nil {
-		for c := range m.KeepCols {
-			if err := checkCol(c, tableCols); err != nil {
-				return errors.Wrap(err, "keep error")
+	// Commenting out to allow keep() to run regardless of existential state of column given
```

Comment:
Can you just delete these lines instead of commenting them out?  Also, is the method `checkCol` just dead code now, not called from anywhere?  If so, it can be deleted too.